fe/fe-core/src/main/java/com/starrocks/common/FeConstants.java

```diff
@@ -39,6 +39,8 @@ public class FeConstants {
 
     public static int checkpoint_interval_second = 60; // 1 minutes
 
+    public static int default_spread_running_task_run_progress_ms = 5000; // 5s
```

Comment:
This configuration may be used in other situations later, so let's give it a more general name.
```suggestion
    public static int sync_task_runs_state_interval = 5000; // 5 seconds
```